source/locale/pt_BR/LC_MESSAGES/internal-communication.po

```diff
@@ -80,674 +80,615 @@ msgstr "IoT agents: agentes para diferentes protocolos."
+"procedure in order to correctly receive all messages."
 msgstr ""
-"Todos os serviços que estão de alguma forma interessados em usar assuntos"
-" devem executar este procedimento para receber corretamente todas as "
-"mensagens."
+"Todos os serviços que estão de alguma forma interessados em usar *subjects* "
+"devem executar este procedimento para receber corretamente todas as mensagens."
 
-#: ../../source/internal-communication.rst:323
+#: ../../source/internal-communication.rst:281
 msgid "Auth + API gateway (Kong)"
 msgstr ""
 
-#: ../../source/internal-communication.rst:325
+#: ../../source/internal-communication.rst:283
 msgid ""
 "Auth is a service deeply connected to Kong. It is responsible for user "
-"management, authentication and authorization. As such, it is invoked by "
-"Kong whenever an request is received by one of its registered endpoints. "
-"This section will detail how this is performed and how they work "
-"together."
+"management, authentication and authorization. As such, it is invoked by Kong "
+"whenever an request is received by one of its registered endpoints. This section "
+"will detail how this is performed and how they work together."
 msgstr ""
 "Auth é um serviço profundamente conectado ao Kong. É responsável pelo "
-"gerenciamento, autenticação e autorização do usuário. Como tal, é "
-"invocado por Kong sempre que uma solicitação é recebida por um de seus "
-"pontos de um dos endpoints. Esta seção detalha como isso é realizado e "
-"como eles funcionam juntos."
+"gerenciamento, autenticação e autorização do usuário. Como tal, é invocado por "
+"Kong sempre que uma solicitação é recebida por um de seus pontos de um dos "
+"*endpoints*. Esta seção detalha como isso é realizado e como eles funcionam "
+"juntos."
 
-#: ../../source/internal-communication.rst:331
+#: ../../source/internal-communication.rst:289
 msgid "Kong configuration"
 msgstr "Configuração do Kong"
 
-#: ../../source/internal-communication.rst:333
+#: ../../source/internal-communication.rst:291
 msgid "There are two configuration procedures when starting Kong within dojot:"
 msgstr "Existem dois procedimentos de configuração ao iniciar o Kong na dojot:"
 
-#: ../../source/internal-communication.rst:335
+#: ../../source/internal-communication.rst:293
 msgid "Migrating existing data"
 msgstr "Migrando Dados Existentes"
 
-#: ../../source/internal-communication.rst:336
+#: ../../source/internal-communication.rst:294
 msgid "Registering API endpoints and plugins."
-msgstr "Registrando endpoints e plug-ins de API."
+msgstr "Registrando endpoints e plugins de API."
 
-#: ../../source/internal-communication.rst:338
+#: ../../source/internal-communication.rst:296
 msgid "The first task is performed by simply invoking Kong with a special flag."
 msgstr ""
-"A primeira tarefa é realizada simplesmente invocando Kong com uma "
-"bandeira especial."
+"A primeira tarefa é realizada simplesmente invocando Kong com uma bandeira "
```

Comment:
A primeira tarefa é realizada simplesmente invocando **o** Kong com uma **flag** especial.

The term `flag` is generally not translated into Portuguese (from what I understand in other documentation)